TrkReco/src/MergeHelices_module.cc

```diff
@@ -183,13 +196,46 @@ namespace mu2e {
     nh2 = shiv2.size();
     nover = countOverlaps(shiv1,shiv2);
   }
-
+	
+  // The chisquared calculation method used here is 
+  // taken from the LsqSums approach followed in CalPatRec
+  void MergeHelices::findchisq(art::Event const& evt,
+      HelixSeed const& h1,float& chixy, float& chizphi) {
+    ::LsqSums4 sxy;
+    ::LsqSums4 szphi;
+    for(size_t ihit=0;ihit < h1.hits().size(); ihit++) {
```

Comment:
Since you never use the 'ihit' variable except to dereference the hit, you can simplify the code by using a range loop:
for (auto const& hh : h1.hits()) {